services/httpd/handler.go

```diff
@@ -464,15 +465,15 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *meta.
 	_, err := buf.ReadFrom(body)
 	if err != nil {
 		if h.Config.WriteTracing {
-			h.Logger.Print("write handler unable to read bytes from request body")
+			h.Logger.Print("Write handler unable to read bytes from request body.")
```

Comment:
I don't know why I added punctuation, I'll remove it. Logged messages though, unlike error messages, should be capitalised.